peachjam_api/urls.py

```diff
@@ -9,4 +9,9 @@
 
 urlpatterns = [
     path("", include(router.urls)),
+    path(
+        "documents/<int:pk>/enrichments/citation-links",
+        views.CitationLinkDetail.as_view(),
```

Comment:
The `CitationLinkDetail` view doesn't apply the `pk` filter to anything, so this view is no different to a normal listing view -- which is fine. Perhaps simply add a `citation-links` endpoint to the `router` above, that is a normal CRUD view? I don't see a reason right now to have two separate views.

Reply:
Done.